lib/attrsets.nix

```
@@ -680,52 +680,50 @@ rec {
   attrsToList = mapAttrsToList nameValuePair;
 
 
-  /* Like `mapAttrs`, except that it recursively applies itself to
-     the *leaf* attributes of a potentially-nested attribute set:
-     the second argument of the function will never be an attrset.
-     Also, the first argument of the argument function is a *list*
-     of the attribute names that form the path to the leaf attribute.
+  /**
+    Like `mapAttrs`, except that it recursively applies itself to the *leaf* attributes of a potentially-nested attribute set:
+    the second argument of the function will never be an attrset.
+    Also, the first argument of the mapping function is a *list* of the attribute names that form the path to the leaf attribute.
 
-     For a function that gives you control over what counts as a leaf,
-     see `mapAttrsRecursiveCond`.
+    For a function that gives you control over what counts as a leaf, see `mapAttrsRecursiveCond`.
 
-     Example:
-       mapAttrsRecursive (path: value: concatStringsSep "-" (path ++ [value]))
-         { n = { a = "A"; m = { b = "B"; c = "C"; }; }; d = "D"; }
-       => { n = { a = "n-a-A"; m = { b = "n-m-b-B"; c = "n-m-c-C"; }; }; d = "d-D"; }
+    Example:
```

Since you moved to RFC145 docstrings, these won't work as you expect anymore: they're just normal text now. To have these as sections again, you'll need to be explicit with the headers and will need to adjust the content of the examples too.

```suggestion
    # Examples
```

---

But will that already render correctly?

---

See https://github.com/NixOS/nixpkgs/pull/292310 for an example of the kind of changes you'll need to make (you can also leave it to @hsjobeki and me since our plan is to migrate all the code in `lib` in batches to the new docstrings).

---

Done